selfdrive/car/isotp_parallel_query.py

```diff
@@ -135,6 +135,7 @@ def get_data(self, timeout, total_timeout=60.):
             if self.debug:
               cloudlog.warning(f"iso-tp query response pending: {tx_addr}")
           else:
+            response_timeouts.pop(tx_addr, None)
```

Comment:
```suggestion
            response_timeouts[tx_addr] = 0
```

Would this make more sense?